packages/augur-ui/src/modules/trading/components/form.styles.less

```diff
@@ -2,7 +2,7 @@
 
 .TradingForm {
   > ul {
-  padding: @size-12 @size-12 0;
```

Comment:
this should be reversed, it's breaking the look of the order ticket (keep it as padding: 12 12 0;)

Reply:
Thank you for pointing out. Reverted the change.